telemetry-collector/tests/cpp/gps.cpp

```diff
@@ -66,6 +66,43 @@ int invalidTest() {
 }
 
 
+int emptyModule() {
```

Comment:
Is the intentional to be lumped in the PR?

Reply:
Oh yeah this was me trying to figure out why GPS is showing all 1's. Might as well lump this test in with this PR